Data/Sys/GameSettings/NAE.ini

```diff
@@ -1,4 +1,4 @@
-# NAEE01 - Paper Mario
+# NAEP01, NAEE01 - Paper Mario
```

Comment:
It's not too important, but these IDs should preferably be sorted alphabetically.